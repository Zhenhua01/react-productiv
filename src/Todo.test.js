import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

describe("basic rendering", function () {


  it("renders without crashing", function () {
    const result = render(<Todo id={1}
      title={"nope"}
      description={"nope description"}
      priority={1} />);

    expect(result.queryByText("nope")).toBeInTheDocument();
  });

  it("matches snapshot", function () {
    const { container } =
      render(
        <Todo id={1}
          title={"nope"}
          description={"nope description"}
          priority={1}

        />
      );
    expect(container).toMatchSnapshot();

  });


});

