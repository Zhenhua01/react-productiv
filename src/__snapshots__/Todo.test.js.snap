// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic rendering matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        nope
      </b>
       
      <small>
        (priority: 
        1
        )
      </small>
    </div>
    <div>
      <small>
        nope description
        .
      </small>
    </div>
  </div>
</div>
`;
